stdlib/asyncio/futures.pyi

```diff
@@ -3,6 +3,7 @@ from _typeshed import Self
 from concurrent.futures._base import Error, Future as _ConcurrentFuture
 from typing import Any, Awaitable, Callable, Generator, Iterable, TypeVar
 
+from .base_futures import isfuture
```

Comment:
```suggestion
from .base_futures import isfuture as isfuture
```

Imported names are considered "private to the stub" unless this weird idiom is used :)